src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -52,23 +52,30 @@
   <!-- This project references the SonarLint TeamExplorer assembly which is VS-version specific -->
   <Import Project="..\SonarLint.VSSpecificAssemblies.props" />
 
-  <!-- Need to reference the Sonar Roslyn analyzers: we need to reference one of the assmblies,
-       and embed all of them in the VSIX. -->
+  <!-- ********************************************************** -->
+  <!-- Sonar C#/VB analyzers -->
+  <!-- ********************************************************** -->
   <Import Project="..\EmbeddedSonarAnalyzer.props" />
   <ItemGroup Label="Embedded SonarLint analyzers">
-    <!-- Reference the packages we want to import, but we don't want to include them as analyzers for project -->
+    <!-- Reference the packages we want to import, but don't include them as analyzers for this project -->
     <PackageReference Include="SonarAnalyzer.CSharp" Version="$(EmbeddedSonarAnalyzerVersion)" ExcludeAssets="all" PrivateAssets="all" />
     <PackageReference Include="SonarAnalyzer.VisualBasic" Version="$(EmbeddedSonarAnalyzerVersion)" ExcludeAssets="all" PrivateAssets="all" />
 
     <!-- Select all of the analyzer binaries (we assume that the package restore has happened so the files are already on disc) -->
     <SonarAnalyzerAssemblies Include="$(NuGetPackageRoot)SonarAnalyzer.CSharp\$(EmbeddedSonarAnalyzerVersion)\analyzers\*.dll" />
     <SonarAnalyzerAssemblies Include="$(NuGetPackageRoot)SonarAnalyzer.VisualBasic\$(EmbeddedSonarAnalyzerVersion)\analyzers\*.dll" />
 
-    <Reference Include="@(SonarAnalyzerAssemblies)" />
-    <!-- need to reference the common SonarAnalyzer.dll assembly directly -> no harm in referencing them all -->
```

Comment:
This is now a problem if we reference them all since we'll pick up the wrong version of the NewtonSoft analyzers.

However, we still need to make sure all of the assemblies are included and the VSIX, and we need them to be in the content folder so we can include them in the AsmRef check.

Reply:
Ah I understand. Ok.